tools/test-npm.sh

```diff
@@ -3,11 +3,11 @@
 set -e
 
 # always change the working directory to the project's root directory
-cd $(dirname $0)/..
+cd $(dirname ${BASH_SOURCE[0]})/..
```

Comment:
Nothing really, I just saw it was suggested in the stackoverflow, going to revert to using `$0`